lib/xcake/target/sugar.rb

```diff
@@ -1,11 +1,17 @@
 require "xcodeproj"
 
 module Xcake
+  def self.strip_heredoc(docs)
+      indent = docs.scan(/^[ \t]*(?=\S)/).min
+      indent_len = (indent || "").length
+      docs.gsub(/^[ \t]{#{indent_len}}/, '')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.